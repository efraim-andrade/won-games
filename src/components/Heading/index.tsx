import React from 'react'

import * as S from './styles'

const Heading: React.FC = () => {
  return (
    <S.Container>
      <h1>component: Heading</h1>
    </S.Container>
  )
}

export default Heading
