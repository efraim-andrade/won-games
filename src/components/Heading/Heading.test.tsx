import { render } from '@testing-library/react'

import Heading from '.'

describe('<Heading />', () => {
  it('should render the Heading', () => {
    const { container } = render(<Heading />)

    expect(container).toMatchSnapshot()
  })
})
