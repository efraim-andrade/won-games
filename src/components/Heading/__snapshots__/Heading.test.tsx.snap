// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> should render the Heading 1`] = `
<div>
  <div
    class=""
  >
    <h1>
      component: Heading
    </h1>
  </div>
</div>
`;
